File: clients/java/sync/examples/Fibonacci.java
// Fibonacci example for unsandbox Java SDK
// Expected output: fib(10) = 55

public class Fibonacci {
    public static int fib(int n) {
        if (n <= 1) return n;
        return fib(n - 1) + fib(n - 2);
    }

    public static void main(String[] args) {
        System.out.println("fib(10) = " + fib(10));
    }
}

File: clients/php/sync/examples/fibonacci.php
#!/usr/bin/env php
<?php
/**
 * Fibonacci example for unsandbox PHP SDK
 * Expected output:
 * fib(10) = 55
 * fib(20) = 6765
 */

function fib(int $n): int {
    if ($n <= 1) {
        return $n;
    }
    return fib($n - 1) + fib($n - 2);
}

echo "fib(10) = " . fib(10) . "\n";
echo "fib(20) = " . fib(20) . "\n";
